### PR TITLE
Bring back python-flup

### PR DESCRIPTION
**Pull request by BKPepe**
Maintainers: @commodo and @me
Compile tested: N/A
Run tested: n/a

Description:
as discussed here https://github.com/openwrt/packages/pull/11797#issuecomment-655411983

I was thinking if in the revert commit I shouldnt add me as maintainer, but there isnt anything wrong with 2 commits? I don't have strong opinion on this one.

### CODE REVIEW

**Review by commodo — APPROVED** [review]

File: lang/python/flup/Makefile
Comment (commodo) [inline lang/python/flup/Makefile:37]: so a few quick things maybe: can you remove prefix `+PACKAGE_python3-flup:`

File: lang/python/flup/Makefile
Comment (commodo) [inline lang/python/flup/Makefile:17]: maybe bump PKG_RELEASE with this change?

Comment (commodo) [reply lang/python/flup/Makefile:17]: i.e. in comit `python-flup: add me as co-maintainer `

File: lang/python/flup/Makefile
Comment (commodo) [inline lang/python/flup/Makefile:25]: maybe also do a quick merge of this Default into the main `define Package/python3-flup`?

**Comment by commodo** [comment]
so this is fine from to merge as-is from my side;
only 1 comment may be a bit more important [about PKG_RELEASE]

rest we can do later;

**Comment by BKPepe** [comment]
I thought that reverting it would be fine, my bad. I will do all the changes as requested. You are right that we should polish it before merging.

**Comment by commodo** [comment]
> I thought that reverting it would be fine, my bad. I will do all the changes as requested. You are right that we should polish it before merging.

i probably removed it sometime after plenty of reworks & removing of Python2 variants happened

**Comment by BKPepe** [comment]
Sorry took me some time during holidays. Now, it should be done. Would you please check it once again? I compile and run tested it on Turris Omnia, OpenWrt master.

**Review by commodo — APPROVED** [review]

**Comment by BKPepe** [comment]
I think, now it is the right time to drop this. :) According to 
https://gitlab.nic.cz/turris/os/packages/-/merge_requests/1355 

cc: @shenek and @aleksan4eg

**Pull request merged**
